modules/core/shared/src/main/scala/gem/Target.scala

```diff
@@ -3,5 +3,23 @@
 
 package gem
 
+import cats.Eq
+
+import gem.enum.Site
+import gem.math.Ephemeris
+
+import monocle.Optional
+import monocle.macros.Lenses
+
 /** A target of observation. */
-final case class Target(name: String, track: Track)
+@Lenses final case class Target(name: String, track: Track)
+
+@SuppressWarnings(Array("org.wartremover.warts.PublicInference"))
+object Target {
+
+  val ephemerides: Optional[Target, Map[Site, Ephemeris]] =
+    track composeOptional Track.ephemerides
```

Comment:
They are kind of long but I think I prefer the `composeX` methods over the corresponding weathervane operators like `^|-?` in the same way that we prefer `foldLeft` over `/:`.

Reply:
Yes, please use `composeX` over `^|-?`. As someone who is trying to keep an eye on OCS3 / gems so as to not fall behind, obscure operators make it much more difficult.

Reply:
If we end up doing a lot of composition of optics we may find that the short versions work out better, but this is fine.